### PR TITLE
Fixed PR "Added banner for mobile"

### PR DESCRIPTION
**Pull request by adriaandotcom**
https://github.com/BustByte/coronastatus/pull/345

### CODE REVIEW

**Review by fossecode — CHANGES_REQUESTED** [review]
There are some US municipality files etc that are not supposed to be in `/locales`

File: README.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ We don't know how many people have COVID-19. So we made a website where people c
 - 🇵🇭 Philippines: coming soon
 - 🇹🇷 Turkey: coming soon (work group Telegram chat: https://t.me/turkeycoronastatus)
 - 🇧🇪 Belgium: coming soon
+- 🇩🇰 Denmark: https://coronastatus.dk
```
Comment (fossecode) [inline README.md]: These three lines should be deleted

File: app/countrySpecific/geonames-converter.py [hunk 2]
```diff
@@ -2,9 +2,9 @@
 from json import dump as write_as_json
 
 # Set the country code you want to create JSON files for
-COUNTRY_CODE = 'bd'
+COUNTRY_CODE = 'en-us'
```
Comment (fossecode) [inline app/countrySpecific/geonames-converter.py]: This can be reverted

**Comment by fossecode** [comment]
The `en-us` locale should be deleted as well

File: app/countrySpecific/geonames-converter.py (hunk 2 above)
Comment (adriaandotcom) [reply app/countrySpecific/geonames-converter.py]: Fixed!

**Comment by adriaandotcom** [comment]
Maybe there are more merge issues here:

<kbd>Screenshot</kbd>

**Comment by fossecode** [comment]
All US files kan be deleted (they are already on master), just add the translation to the existing ones

**Comment by adriaandotcom** [comment]
> All US files kan be deleted (they are already on master), just add the translation to the existing ones

This is with master merged. Does this look okay to you?

<kbd>Screenshot</kbd>

**Review by fossecode — APPROVED** [review]

**Review by fossecode — APPROVED** [review]

**Comment by adriaandotcom** [comment]
Fixed spacing on mobile as well @amritnagi 

<kbd>Screenshot</kbd>

**Pull request merged**